File: DevHiringChallenge.Domain/Entities/Owner.cs
using System;

namespace DevHiringChallenge.Domain.Entities
{
    public class Owner
    {
        public Owner(string login, int id, string nodeId, string avatarUrl, string type, string url)
        {
            Login = login;
            Id = id;
            Node_Id = nodeId;
            Avatar_Url = avatarUrl;
            Type = type;
            Url = url;
        }

        public Guid CodigoOwner { get; set; }
        public string Login { get; set; }
        public int Id { get; set; }
        public string Node_Id { get; set; }
        public string Avatar_Url { get; set; }
        public string Type { get; set; }
        public string Url { get; set; }
    }
}

File: back-end/src/main/java/com/github/integration/backend/persistence/entities/Owner.java
package com.github.integration.backend.persistence.entities;

import lombok.*;

import javax.persistence.Entity;
import javax.persistence.Id;

@NoArgsConstructor
@AllArgsConstructor
@Builder
@Getter
@Setter
@Entity
public class Owner {

    private String login;

    @Id
    private Long id;

    private String nodeId;

    private String avatarUrl;

    private String gravatarId;

    private String url;

    private String htmlUrl;

    private String followersUrl;

    private String followingUrl;

    private String gistsUrl;

    private String starredUrl;

    private String subscriptionsUrl;

    private String organizationsUrl;

    private String reposUrl;

    private String eventsUrl;

    private String receivedEventsUrl;

    private String type;

    private Boolean siteAdmin;

}